### ISSUE TITLE
Unify covariate shift ID splits

### ISSUE THREAD

**Issue by zjysteven** [OpenOOD/openood#7]
Currently the `csid` for CIFAR-10/100 is CIFAR-10/100c (corrupted CIFAR images), while the one for ImageNet is ImageNetV2 (clean images).

**Comment by Jingkang50** [comment]
we also have cinic-10 and imagenet-10 datasets to support cifat-10's csid.
but I dont have any more good idea for cifar-100's csid.

**Comment by zjysteven** [comment]
Right, CIFAR-100 indeed lacks these types of covariate-shift datasets:

| | CIFAR-10 | CIFAR-100 | ImageNet |
|:----|:-----:|:----:|:-----:|
| Common corruptions | CIFAR-10-C | CIFAR-100-C | ImageNet-C |
| Dataset resampling bias | CIFAR-10.1 | ? | ImageNetV2 |
| Image source shift / Subpopulation robustness | CINIC-10, ImageNet-10 | ? | ? |
| Style shift | ? | ? | ImageNet-R, ImageNet-Sketch | 

I was thinking maybe we could just focus on CIFAR-10 and ImageNet for evaluating full-spectrum OOD. I do think that touching more on OOD generalization v.s. OOD detection is important and could be one of the big selling points for OpenOODv2.

**Comment by Jingkang50** [comment]
agree. actually by only exploring FS-OOD on imagenet dataset is meaningful enough. Considering cifar-10 is a plus.
